pkg/controller/controller_binding.go

```diff
@@ -289,26 +320,36 @@ func (c *controller) reconcileBinding(binding *v1alpha1.Binding) error {
 				err,
 			)
 			glog.Warning(s)
-			c.updateBindingCondition(
+			if err := c.updateBindingCondition(
 				binding,
 				v1alpha1.BindingConditionReady,
 				v1alpha1.ConditionFalse,
 				errorUnbindCallReason,
-				"Unbind call failed. "+s)
+				"Unbind call failed. "+s,
+			); err != nil {
+				glog.Errorf("updating binding condition (%s)", err)
+				return err
+			}
 			c.recorder.Event(binding, api.EventTypeWarning, errorUnbindCallReason, s)
 			return err
 		}
 
-		c.updateBindingCondition(
+		if err := c.updateBindingCondition(
 			binding,
 			v1alpha1.BindingConditionReady,
 			v1alpha1.ConditionFalse,
 			successUnboundReason,
 			"The binding was deleted successfully",
-		)
+		); err != nil {
+			glog.Errorf("updating binding condition (%s)", err)
+			return err
+		}
 		// Clear the finalizer
 		finalizers.Delete(v1alpha1.FinalizerServiceCatalog)
-		c.updateBindingFinalizers(binding, finalizers.List())
+		if err := c.updateBindingFinalizers(binding, finalizers.List()); err != nil {
```

Comment:
So for example here, I think this is more readable using this format because things can fit in a single line :)